client/src/components/history-timeline.tsx

```
@@ -0,0 +1,509 @@
+    if (diffMins < 60) return t('history.minutesAgo', `${diffMins}m ago`);
+    if (diffHours < 24) return t('history.hoursAgo', `${diffHours}h ago`);
+    if (diffDays < 7) return t('history.daysAgo', `${diffDays}d ago`);
+    return date.toLocaleDateString();
+  };
+
+  const getPlatformIcon = (platform: string) => {
+    switch (platform.toLowerCase()) {
+      case 'spotify':
+        return <SiSpotify className="h-4 w-4 sm:h-5 sm:w-5 text-spotify" />;
+      case 'youtube':
+        return <SiYoutubemusic className="h-4 w-4 sm:h-5 sm:w-5 text-youtube" />;
```

ESLint

[error] Replace `<SiYoutubemusic·className="h-4·w-4·sm:h-5·sm:w-5·text-youtube"·/>` with `(⏎··········<SiYoutubemusic·className="h-4·w-4·sm:h-5·sm:w-5·text-youtube"·/>⏎········)`

(prettier/prettier)